src/Web/Avalonia.Web/BrowserSingleViewLifetime.cs

```diff
@@ -47,6 +47,13 @@ public static class WebAppBuilder
         where T : AppBuilderBase<T>, new()
     {
         return builder
+            .AfterSetup(_ =>
```

Comment:
Why are you using `AfterSetup` instead of registering it in `BrowserWindowingPlatform.Register` ?

Reply:
@danwalmsley

Reply:
@kekekeks @maxkatz6 it gets overwritten otherwise.

Reply:
Fixed too